tests/ui/diagnostic_namespace/do_not_recommend/supress_suggestions_in_help.next.stderr

```diff
@@ -12,6 +12,10 @@ note: required by a bound in `check`
    |
 LL | fn check(a: impl Foo) {}
    |                  ^^^ required by this bound in `check`
+help: use a unary tuple instead
+   |
+LL |     check(((),));
+   |           +  ++
```

Comment:
Is this suggestion okay? I'm not quite sure what `#[do_not_recommend]` is supposed to (not) show. Its rfc only talks about not mentioning confusing traits.

Reply:
I think it's fine.